kubectl/kubectl-deploy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: kubectl-deploy-pod
+  namespace: default
```

Comment:
maybe we should not define static namespace?

Reply:
Since the `rolebinding` is for the `default` namespace